composer.json

```diff
@@ -27,12 +27,12 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
-    "guzzlehttp/guzzle": "~6.0@dev|>=6",
-    "psr/log": "~1|~1.0|1.0.0|dev-master"
+    "php": "^5.5.9 || ^7.0",
```

Comment:
that's fine - i just tend to do the opposite, just like phpunit is doing, and forcing people to secure their systems